### ISSUE TITLE
[Suggestion] Some way to return Spirit Tree Groves to their former glory (or just some sort of sludge plains remover)

### ISSUE THREAD

**Issue by CrazyAceking** [TheCyberBrick/The-Betweenlands#1247]
TLDR: I'd love some way to remove the Sludge biome that a dead Spirit Tree left behind.

So, I made my base in a giant weedwood tree, which just so happened to be very close to a Spirit Tree grove. I promptly killed the tree because it was a nuisance that kept firing at me if I even tried working on my base nearby. Of course, my illegal logging led to the creation of a mini Sludge biome right at my doorstep, which is a nice form of generating Sludge and spawning Wights, sure, but it's dreadfully ugly. I'd love some sort of way to atone for my sins and return the area to its former glory. Or at the very least, some form of way to convert it from being a sludge plains. Maybe do what Thaumcraft did with Silverwood trees, where planting them converts the nearby biome into Magical Forest. Maybe planting Spirit Trees in the Sludge Plains could convert the area around the trees back into Grottos.

**Comment by PetasiGit** [comment]
If you remove the sludge blocks when the tree is first killed you can prevent it from spreading to the entire island. I know that's not exactly removing it, but there _is_ a way to prevent it. I think it's unlikely that there will ever be a way added to undo decay/corruption/destruction like that. Forces of nature can't always be controlled, ya know?

**Comment by vadis365** [comment]
Thank you for your suggestion - it has been closed, but not forgotten, in order for us to be able to manage our issue tracker for bugs.
